README.md

```diff
@@ -7,100 +7,78 @@
 
-To simplify installation and use, the tool is available as a Docker image.
+Included in this module is [Kubetool](https://github.com/puppetlabs/puppetlabs-kubernetes/blob/master/tooling/kube_tool.rb), a configuration tool that auto-generates the Hiera security parameters, the discovery token hash, and other configurations for your Kubernetes cluster. To simplify installation and use, the tool is available as a Docker image.
 
-### Generate the module configuration
+### Generating the module configuration
 
 If Docker is not installed on your workstation, install it from [here](https://docs.docker.com/install/).
 
 The Kubetool Docker image takes each parameter as an environment variable.
 
-**Note**: The version of Kubetool you use must match the version of the module on the Puppet Forge. For example, if using the module version 1.0.0, use `puppet/kubetool:1.0.0`.
+**Note:**: The version of Kubetool you use must match the version of the module on the Puppet Forge. For example, if using the module version 1.0.0, use `puppet/kubetool:1.0.0`.
 
 To output a yaml file into your working directory that corresponds to the operating system you want Kubernetes to run on, and for each controller node, run either of these `docker run` commands:
 
 ```
 docker run --rm -v $(pwd):/mnt --env-file .env puppet/kubetool:{$module_version}
 ```
 
-The docker run command above includes a .env file which is included in the root folder of this repo.
+The `docker run` command above includes a `.env` file which is included in the root folder of this repo.
 
 ```
 docker run --rm -v $(pwd):/mnt -e OS=debian -e VERSION=1.10.2 -e CONTAINER_RUNTIME=docker -e CNI_PROVIDER=weave -e ETCD_INITIAL_CLUSTER=kube-master:172.17.10.101,kube-replica-master-01:172.17.10.210,kube-replica-master-02:172.17.10.220 -e ETCD_IP="%{::ipaddress_eth1}" -e KUBE_API_ADVERTISE_ADDRESS="%{::ipaddress_eth1}" -e INSTALL_DASHBOARD=true puppet/kubetool:{$module-version}
 ```
 
-The parameters are:
+The above parameters are:
 
-* `OS`: The operating system Kubernetes runs on.
-* `VERSION`: The version of Kubernetes to deploy.
-* `CONTAINER_RUNTIME`: The container runtime Kubernetes uses. Set this value to `docker` or `cri_containerd`.
-* `CNI_PROVIDER`: The CNI network to install. Set this value to `weave` or `flannel`.
-* `ETCD_INITIAL_CLUSTER`: The server hostnames and IPs in the form of `hostname:ip`. When in production, include three, five, or seven nodes for etcd.
-* `ETCD_IP`: The IP each etcd member listens on. We recommend passing the fact for the interface to be used by the cluster.
-* `KUBE_API_ADVERTISE_ADDRESS`: The IP each etcd/apiserver instance uses on each controller. We recommend passing the fact for the interface to be used by the cluster.
-* `INSTALL_DASHBOARD`: A boolean which specifies whether to install the dashboard.
+* `OS`: The operating system Kubernetes runs on. 
```

Comment:
Wee side note, I'd remove all the extra spaces at the end of these sentences, they can trip up the translator so you'd be saving yourself work later on if the module ever got translated.

Reply:
Thanks @HelenCampbell for pointing this out.